mmv1/third_party/terraform/services/container/resource_container_cluster.go.tmpl

```diff
@@ -6983,6 +7043,19 @@ func flattenWorkloadAltsConfig(c *container.WorkloadALTSConfig) []map[string]int
 }
 {{- end }}
 
+{{ if ne $.TargetVersionName `ga` -}}
+func flattenPodAutoscaling(c *container.PodAutoscaling) []map[string]interface{} {
```

Comment:
If the API is returning UNSPECIFIED when an empty string is sent (and this behavior is consistent - that is, doesn't change based on other factors) you can intercept here & convert it to an empty string rather than using a diff suppress; generally speaking, it's better to make that kind of change in a flattener if possible.